### PR TITLE
Increase test matrix for smoke tests

### PR DESCRIPTION
**Pull request by evrardjp**
This increases the test matrix for the kind smoke tests.

### CODE REVIEW

**Comment by evrardjp** [comment]
Do not merge until https://github.com/weaveworks/kured/pull/183 has merged.

**Review by ckotzbauer — APPROVED** [review]

File: .github/workflows/smoke-tests.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          EOF
+
+      - name: Create 5 node kind cluster
+        uses: helm/kind-action@master
+        with:
+          config: .github/kind-cluster-${{ matrix.kubernetes-version}}.yaml
+
+      - name: Deploy kured on default namespace with its helm chart
+        run: |
+          curl -o helm3.tgz https://get.helm.sh/helm-v3.3.0-linux-amd64.tar.gz
```
Comment (dholbach) [inline .github/workflows/smoke-tests.yml:48]: Update?

Comment (dholbach) [reply .github/workflows/smoke-tests.yml:48]: Duplicated, right?

Comment (evrardjp) [reply .github/workflows/smoke-tests.yml]: It is in one of dependent PRs which will disappear on rebase.

**Comment by evrardjp** [comment]
Rebased, now that 183 has merged.

**Review by dholbach — APPROVED** [review]

**Pull request merged**